airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/BaseDatabaseConfigPersistenceTest.java

```diff
@@ -55,15 +57,37 @@ public static void dbDown() {
 
   static {
     try {
-      final ConfigPersistence seedPersistence = YamlSeedConfigPersistence.getDefault();
-      SOURCE_GITHUB = seedPersistence
-          .getConfig(ConfigSchema.STANDARD_SOURCE_DEFINITION, "ef69ef6e-aa7f-4af1-a01d-ef775033524e", StandardSourceDefinition.class);
-      SOURCE_POSTGRES = seedPersistence
-          .getConfig(ConfigSchema.STANDARD_SOURCE_DEFINITION, "decd338e-5647-4c0b-adf4-da0e75f5a750", StandardSourceDefinition.class);
-      DESTINATION_SNOWFLAKE = seedPersistence
-          .getConfig(ConfigSchema.STANDARD_DESTINATION_DEFINITION, "424892c4-daac-4491-b35d-c6688ba547ba", StandardDestinationDefinition.class);
-      DESTINATION_S3 = seedPersistence
-          .getConfig(ConfigSchema.STANDARD_DESTINATION_DEFINITION, "4816b78f-1489-44c1-9060-4b19d5fa9362", StandardDestinationDefinition.class);
+      SOURCE_GITHUB = new StandardSourceDefinition()
```

Comment:
Had to replace these with hardcoded definitions, so that `airbyte-config:persistence` does not have a dependency on `airbyte-config:init`, but I think hardcoding these in this test provides better isolation of these tests anyway.

Reply:
(also I realized that the static bloc surrounding this is now unnecessary, and was removed by this commit)